examples/example_2_research_group_report.py

```diff
@@ -15,8 +17,12 @@
 
     for researcher in researchers:
         s.add_collector(CollectByAuthorID(author_id=researcher))
-    
+
     s.add_filter(FilterByYear(min_year=2022, max_year=2022))
-    
-    for paper in s.iterate( remove_source_from_results=False):
+
+    history_cb = HistoryCallbacks()
+
+    for paper in s.iterate(remove_source_from_results=False, callbacks=[history_cb]):
         print(f"{paper.year}@{paper.paper_id} - {paper.title} => {paper.authors}")
```

Comment:
Why print every paper? Won't History Callbacks print this information in the `dump()`?

Reply:
Yap, I'll remove that.